signinapp/model.py

```diff
@@ -26,6 +27,9 @@
     normalize_phone_number_from_storage,
 )
 
+FINANCE_OVERHEAD = 0.15
```

Comment:
This should be stored per event so we can account for actual costs of running an event.